### PR TITLE
[REF] install: pin coveralls version and supporting broken_uninstallable module

### PR DESCRIPTION
**Pull request by moylop260**
Related with https://github.com/OCA/maintainer-quality-tools/issues/626
cc @Yajo 

And Supporting broken_uninstallable module
After new logger.error from https://github.com/odoo/odoo/commit/c5d6a39

### CODE REVIEW

**Comment by Yajo** [comment]
So, do you mean that the culprit is the version of coveralls?

File: sample_files/.gitlab-ci-2.yml [hunk 1]
```diff
@@ -22,7 +22,6 @@ variables:
 before_script:
   - curl -sL https://deb.nodesource.com/setup_10.x | bash -
   - apt-get install -y  sudo postgresql-client expect-dev python-lxml nodejs python-dev python-pip build-essential libsasl2-dev python-dev libldap2-dev libssl-dev
-  - pip install coverage coveralls codecov
```
Comment (pedrobaeza) [inline sample_files/.gitlab-ci-2.yml]: Shouldn't this install also pinned versions?

**Comment by moylop260** [comment]
> @Yajo 
So, do you mean that the culprit is the version of coveralls?

I saw something related and I just tested pin it

But I didn't looking for more details.

Maybe a recent changes incompatible.

**Comment by pedrobaeza** [comment]
Wasn't this green previously?

**Comment by moylop260** [comment]
Yes, the coverage error was fixed but at the same time a new change was introduced in Odoo to log an new error:
 - https://github.com/odoo/odoo/commit/c5d6a3977de85fb974a4940fb11d54ef847e08e4#diff-2f47c68d06883cfb9f76e474abca7676

So, the new red is unrelated.

**Comment by moylop260** [comment]
Now it is green... until now 😄

**Comment by pedrobaeza** [comment]
Hehe, let's merge it before turning again :yum:

**Pull request merged**